README.md

```diff
@@ -1,6 +1,12 @@
 # MangaJet
 Manga reader project, SPbSTU 2022.
 
+### Product description
+Manga reader is an app for reading manga. Our app aims to make reading manga easier. The app will be an aggregator of several sites that will be parsed, so we get manga. In addition to the reading function, the app will also save your reading history and get updates from the manga you're reading, as well as save selected chapters.
+
+### Description of advantages over peers
+The main difference between this application and its counterparts is the system of recommendations based on the read, authorization and a more beautiful design.
```

Comment:
- [ ] Please define specific examples and do more detailed comparison
What is "more beautiful design"? It's just a matter of taste, isn't it? Thoughtful design? Modern design?